src/pages/graphql/schema/attributes/queries/custom-attribute-metadata-v2.md

```diff
@@ -0,0 +1,128 @@
+---
+title: customAttributeMetadataV2 query | Commerce Web APIs
+---
+
+# customAttributeMetadataV2 query
+
+The `customAttributeMetadataV2` query retrieves metadata for the specified `entity_type` and `attribute_code` pairs.
+
+This query replaces [`attributesMetadata`](../../store/queries/attributes-metadata.md) 
+and [`customAttributeMetadata`](../../store/queries/custom-attribute-metadata.md). 
+
+Some of the differences of this new query are:
+- Can be extended by all entity types. 
+- This provides access to all properties of eav attributes.
+- It introduced uid information for the attributes.
```

Comment:
```suggestion
-   `uid` values are now available.

```

Add a blank line after the last bullet. Also make sure there are no blank spaces at the end any of the lines.